packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -116,6 +116,14 @@ export interface RestApiProps extends ResourceOptions {
    */
   readonly endpointTypes?: EndpointType[];
 
+  /**
+   * A list of the VPC Endpoint IDs of the API. Use this property when creating
+   * a private API.
+   *
```

Comment:
A couple of sentences of what behaviour this property enables is always useful. 

A good technique is to transcribe a couple of sentences from [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/associate-private-api-with-vpc-endpoint.html) and then link to it with a `@see` tag.

Reply:
I added the link and brief explanation that it creates DNS alias.